### PR TITLE
PIM-5159: Attribute values appearing/disappearing when you change attribute groups

### PR DESCRIPTION
**Pull request by grena**
| Q                 | A
| ----------------- | ---
| Specs             | N
| Behats            | N
| Blue CI           |
| Changelog updated | Y
| Review and 2 GTM  |

### CODE REVIEW

**Comment by sumbobyboys** [comment]
GTM

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/multi-select-field.js [hunk 1]
```diff
@@ -64,14 +66,18 @@ define(
                             }
                         },
                         initSelection: function (element, callback) {
-                            $.ajax(choiceUrl).then(function (response) {
+                            if (null === this.choicePromise) {
+                                this.choicePromise = $.ajax(choiceUrl);
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/multi-select-field.js]: `$.get()` ?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/simple-select-field.js [hunk 2]
```diff
@@ -65,12 +67,16 @@ define(
                         initSelection: function (element, callback) {
                             var id = $(element).val();
                             if ('' !== id) {
-                                $.ajax(choiceUrl).then(function (response) {
+                                if (null === this.choicePromise) {
+                                    this.choicePromise = $.ajax(choiceUrl);
```
Comment (phaseinducer) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/product/field/simple-select-field.js]: same

**Comment by phaseinducer** [comment]
jsdoc please =)

**Comment by phaseinducer** [comment]
GTM !

**Pull request merged**